src/mcom.md

```diff
@@ -0,0 +1,25 @@
+---
+ee_only: true
+title: MCOM
+group: operations
+sections:
+  - title: Manage Stock Aggregates
+    content: This category includes information about adding, editing, and managing stock aggregates for your OMS.
```

Comment:
category? Seems like an odd term to use. Do we mean section or page?

Reply:
I was copying what was already there for the top-level Reports page. https://docs.magento.com/user-guide/reports.html

Reply:
Will revise, but you should consider changing the Reports page as well.